Detector/DetComponents/src/MaterialScan.cpp

```diff
@@ -17,6 +17,7 @@ MaterialScan::MaterialScan(const std::string& name, ISvcLocator* svcLoc) : Servi
   declareProperty("filename", m_filename, "file name to save the tree to");
   declareProperty("etaBinning", m_etaBinning, "eta bin size");
   declareProperty("etaMax", m_etaMax, "maximum eta value");
+  declareProperty("nPhiTrials", m_nPhiTrials, "number of random phi values to average over");
```

Comment:
I seem to have missed these properties, when I moved everything to the new Gaudi. Can you do it (I can also do it after we merge)?